api_server/src/http_service.rs

```diff
@@ -1209,8 +1223,8 @@ mod tests {
 
         // Error Case: Invalid Path.
         assert!(
-            parse_netif_req(path, Method::Patch, &body,)
-                == Err(Error::InvalidPathMethod(path, Method::Patch))
+            parse_netif_req(path, Method::Options, &body,)
+                == Err(Error::InvalidPathMethod(path, Method::Options))
```

Comment:
Probably this enum variant should be InvalidPathOrMethod. Or for that matter there should be 2 variants: `InvalidPath` and `InvalidMethod`.